### PR TITLE
Changed event type, fixed fallback option fixes issue #1980

### PR DESCRIPTION
**Pull request by marcellourbani**
@christianguenter2 
I changed 2 things in javascript:

1. replaced 

`oEvent.key || oEvent.keyCode`
with
`oEvent.key || String.fromCharCode( oEvent.keyCode)`

as the key code is numeric and we want the character value

2. replaced event 'keydown' with 'keypress', as in keydown I get the wrong keycode (i.e. 65 for 'a', converts to 'A' )

### CODE REVIEW

**Comment by larshp** [comment]
#1980

File: src/zabapgit_js_common.w3mi.xml [hunk 1]
```diff
@@ -15,13 +15,13 @@
      <RELID>MI</RELID>
      <OBJID>ZABAPGIT_JS_COMMON</OBJID>
      <NAME>filename</NAME>
-     <VALUE>common.js</VALUE>
+     <VALUE>~wwwtmp.js</VALUE>
```
Comment (christianguenter2) [inline src/zabapgit_js_common.w3mi.xml]: Maybe we should exclude this change?

File: src/zabapgit_js_common.w3mi.xml [hunk 2]
```diff
@@ -15,13 +15,13 @@
...
     </WWWPARAMS>
     <WWWPARAMS>
      <RELID>MI</RELID>
      <OBJID>ZABAPGIT_JS_COMMON</OBJID>
      <NAME>mimetype</NAME>
-     <VALUE>text/javascript</VALUE>
+     <VALUE>application/javascript</VALUE>
```
Comment (christianguenter2) [inline src/zabapgit_js_common.w3mi.xml]: Maybe we should exclude this change?

File: src/zabapgit_js_common.w3mi.xml (hunk 1 above)
Comment (marcellourbani) [reply src/zabapgit_js_common.w3mi.xml]: indeed.
I didn't even notice it!

File: src/zabapgit_js_common.w3mi.xml (hunk 2 above)
Comment (marcellourbani) [reply src/zabapgit_js_common.w3mi.xml]: as above.
It's a side effect of editing the js file

File: src/zabapgit_js_common.w3mi.xml (hunk 1 above)
Comment (marcellourbani) [reply src/zabapgit_js_common.w3mi.xml]: fixed (both)

**Pull request merged**